toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1285,6 +1401,21 @@ func copyAdditionalFiles(installChroot *safechroot.Chroot, config configuration.
 	return
 }
 
+// cleanupRpmDatabase removes RPM database if the image does not require a package manager.
+// rootPrefix is prepended to the RPM database path - useful when RPM database resides in a chroot and cleanupRpmDatabase can't be called from within the chroot.
+func cleanupRpmDatabase(rootPrefix string) (err error) {
+	logger.Log.Info("Attempting RPM database cleanup...")
+	rpmDir := strings.Join([]string{rootPrefix, rpmDependenciesDirectory}, "")
```

Comment:
Can `filepath.Join` be used here instead?

Reply:
It can, thanks!